extension/diffheader.js

```diff
@@ -21,6 +21,46 @@ window.diffFileHeader = (() => {
 		};
 	})();
 
+	const maxPixelsAvailable = () => {
+		// Unfortunately can't cache this value, as it'll change with the browsers zoom level
+		const filenameLeftOffset = $('.diff-toolbar-filename').get(0).getBoundingClientRect().left;
+		const diffStatLeftOffset = $('.diff-toolbar-filename + .right').get(0).getBoundingClientRect().left;
+
+		return diffStatLeftOffset - filenameLeftOffset;
+	};
+
+	const parseFileDetails = filename => {
+		const folderCount = (filename.match(/\//g) || []).length;
+		const [, basename] = (filename.match(/(?:\/)([\w\.-]+)$/) || []);
+		const [, topDir] = (filename.match(/^([\w\.-]+)\//) || []);
+
+		return {
+			folderCount,
+			basename,
+			topDir
+		};
+	};
+
+	const updateFileLabel = val => {
+		const target = $('.diff-toolbar-filename').get(0);
+		target.classList.add('filename-height-check');
```

Comment:
LOL! 
![seth-macfarlane-meesaa-T82JkKa5ji168](https://camo.githubusercontent.com/1d4d9cdcf33f45f14d8b78bc5517d62f3f3cba66/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f5438324a6b4b61356a693136382f67697068792e676966)

Update on it's way